src/item.cpp

```diff
@@ -6003,7 +6003,11 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
     }
 
     if( has_var( "NANOFAB_ITEM_ID" ) ) {
-        tagtext += string_format( " (%s)", nname( itype_id( get_var( "NANOFAB_ITEM_ID" ) ) ) );
+        if( has_flag( flag_NANOFAB_TEMPLATE_SINGLE_USE ) ) {
+            tagtext += string_format( " (SINGLE USE %s)", nname( itype_id( get_var( "NANOFAB_ITEM_ID" ) ) ) );
```

Comment:
```suggestion
            //~ Single-use descriptor for nanofab templates. %s = name of resulting item. The leading space is intentional.
            tagtext += string_format( _( " (SINGLE USE %s)" ), nname( itype_id( get_var( "NANOFAB_ITEM_ID" ) ) ) );
```
Should probably mark that for translation since the string is user-facing and has some English words in it

Reply:
Oops, looks like astyle didn't like the longer line :P